docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md

```diff
@@ -4,13 +4,19 @@ author: Abhirup Roy
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Made minor changes in the backend to facillitate the fetching of PDF files.
+- [x] Optimized the PDF data fetching and retrieval process. 
 
 ## Screenshots / Videos 
 
+<video controls width="100%">
+  <source src="/Digital Learners Passbook Updated.mp4"/>
+</video>
+
 ## Contributions
 
+https://github.com/abhirupr123/digital-learners-passbook/commit/b464d0c0570bd318b14d8a8e9500b7e72db68792
```

Comment:
keep all the contributions in points

Reply:
> keep all the contributions in points

Made all the changes

Reply:
> > keep all the contributions in points
> 
> Made all the changes

https://github.com/Code4GovTech/c4gt-milestones/pull/600/commits/3c8c0209c04ac60bc5d98290c56599de09722617